### ISSUE TITLE
Determine modular scale

### ISSUE THREAD

**Issue by skoolbus39** [d-c-n/dcf#18]
What is currently in place is loosely based on the powers of two, as described in Brent Jackson's article, [Mathematical Web Typography](http://jxnblk.com/writing/posts/mathematical-web-typography/). The powers-of-two scale produces easy-to-understand numbers (e.g. .25, .5, 1, 2, 4).

We have a few other options to consider:
UNL currently uses a [perfect fourth modular scale](http://www.modularscale.com/?1&em&1.333). Common aspect ratios for photos and videos align nicely with this scale—16÷9=1.777 and 4÷3=1.333.

I've used the [golden ratio modular scale](http://www.modularscale.com/?1&em&1.618) in several projects and have found it to work great.

Some might say that the perfect fourth and golden ratio scales produce [magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29). However, if we're asking framework users to choose from a set of utility classes with pre-determined values for sizing and spacing, is it important to know that the value of a particular class is 1.5 vs 1.777 vs 1.618 when choosing which class to use is relative only to the other values in the set?

**Comment by Shaderpixel** [comment]
Hello everyone, Richard here.

From my understanding of today's (Oct 26) discussion, participants have voiced that font-size values being abstracted into class names might play a more importance role than the actual value itself for the typical framework user. I would agree with that line of thought. I have, at times, apply existing classes to see if they achieve my desired visual effect, and only after I have exhausted my classname options do I specify my own font-size. For the advanced devs, nothing is as easy as going to a website or printing out the scale and sticking it somewhere visible. I have found that after using the same font-sizes couple of times, it just sticks in my head. 

I have suggested to @skoolbus39 that he might as well go ahead and try a scale which matches his needs and doesn't have huge negative impact on the existing design that he has created so far. In addition, it be easier on us to judge visually if one scale is superior than the other. 

Overall, I think there is quite a number of design factors that hinges on the chosen scale and we should move quickly on deciding one so that if one scale doesn't work, we have enough time to decide on another.
